Composer/packages/client/src/components/WebChat/utils/conversationService.ts

```diff
@@ -63,6 +64,7 @@ export type ChatData = {
   projectId: string;
   user: User;
   conversationId: string;
+  webChatStore: any;
```

Comment:
do you have a type for this?

Reply:
We dont. Webchat package does not provide types for its store. Let me using unknown